src/Ocelot/Requester/MessageInvokerPool.cs

```diff
@@ -31,11 +33,9 @@ public HttpMessageInvoker Get(DownstreamRoute downstreamRoute)
     }
 
     public void Clear() => _handlersPool.Clear();
-
-    /// <summary>
-    /// TODO This should be configurable and available as global config parameter in ocelot.json.
-    /// </summary>
+    
     public const int DefaultRequestTimeoutSeconds = 90;
```

Comment:
We could probably remove the `DefaultRequestTimeoutSeconds` from `MessageInvokerPool`

Reply:
I agree with this movement. We need to reference to the value of new shared class.
`GlobalPolicy`?

Reply:
#### Update 2
I overlooked the QoS user scenarios... Our existing logic uses two different constants. On the other hand, should the behavior of a standard route request (with a 90-second timeout) differ from the QoS default value? I realized today that the 90-second timeout for both traditional forwarding (without QoS) and QoS-enabled routes might need different values. A normal route without QoS is stable, so 90 seconds is acceptable. However, a QoS-enabled route is less stable due to high load, suggesting the timeout should be shorter. Since we define `TimeoutValue` in most cases, it could be reduced by two to three times.
@RaynaldM, what do you think? Should we maintain the 90-second default value for QoS when timeouts aren't specified? Although, this seems to be a very rare case.